chest/core.py

```diff
@@ -38,14 +42,33 @@ def _do_nothing(*args, **kwargs):
     pass
 
 
+class full_policy(object):
+    """ Behaviours to use when a a chest is full.
+    """
+    @staticmethod
+    def raise_(chest):
+        """A policy that raises an ENOSPC error if the disk is full.
+        """
+        raise OSError(errno.ENOSPC, os.strerror(errno.ENOSPC), chest.path)
+
+    @staticmethod
+    def pop_lru(chest):
+        """A policy that removes the most recently used element from
+        the cache when the chest is full.
+        """
+        del chest[chest.heap.popitem()[0]]
```

Comment:
I think I would prefer a submodule rather than a class.  I would normally wait to produce a submodule until I had a larger critical mass of functions.  Not a big deal either way though.